lib/cloudregister/registerutils.py

```diff
@@ -378,8 +503,13 @@ def get_instance_data(config):
                     logging.error(errMsg)
 
     if instance_data:
+        # Marker for the server to not return https:// formated
+        # service and repo information
+        instance_data += b'<repoformat>plugin:susecloud</repoformat>\n'
         return instance_data.decode()
 
+    return instance_data
```

Comment:
Seems like `.decode()` got lost here. Not sure what encoding it has. Also now there are two `return instance_data` statements -- one should suffice.